update advanced api flow example (#1175)
grafana/k6-docs: src/data/markdown/docs/05 Examples/01 Examples/10 api-crud-operations.md

---
title: 'API CRUD Operations'
excerpt: 'This example covers the usage of k6 to test a REST API CRUD operations.'
---

The examples showcase the testing of CRUD operations on a REST API. 

CRUD refers to the basic operations in a database: Create, Read, Update, and Delete. We can map these operations to HTTP methods in REST APIs:

- _Create_: HTTP `POST` operation to create a new resource.
- _Read_: HTTP `GET` to retrieve a resource.
- _Update_: HTTP `PUT`or `PATCH` to change an existing resource.
- _Delete_: HTTP `DELETE` to remove a resource.

> You'll find two test examples: the first one uses the core k6 APIs (`k6/http` and `checks`), and the second showcases more recent APIs (`httpx` and `k6chaijs`).

Reviewer: ```suggestion
This document has two examples, one that uses the core k6 APIs (`k6/http` and `checks`) and another to show the more recent APIs (`httpx` and `k6chaijs`).
```
